content/courses/native-onchain-development/paging-ordering-filtering-data-frontend.md

```diff
@@ -162,21 +171,23 @@ For example, you could search through a list of contacts by including a `memcmp`
 filter:
 
 ```tsx
+const DATA_OFFSET = 2; // The offset into the program account data - Skip the first 2 bytes of metadata
+const DATA_LENGTH = 18; // The length of the data slice to retrieve - Retrieve 18 bytes of relevant data starting from the 3rd byte
 async function fetchMatchingContactAccounts(
   connection: web3.Connection,
   search: string,
 ): Promise<(web3.AccountInfo<Buffer> | null)[]> {
   const accounts = (await connection.getProgramAccounts(
     new PublicKey(MOVIE_REVIEW_PROGRAM_ID),
     {
-      dataSlice: { offset: 2, length: 18 },
+      dataSlice: { offset: DATA_OFFSET, length: DATA_LENGTH },
       filters:
         search === ""
           ? []
           : [
               {
                 memcmp: {
-                  offset: 6,
+                  offset: 6, // Compare starting from the 7th byte
```

Comment:
OK but why the 7th byte? 

(Don't answer, just put the reason into the code or comment)